client/src/Components/CommonComponent/Menu/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import './index.css';
 import { BrowserRouter as Router, NavLink } from 'react-router-dom';
 
-const Menu = ({ showOrHide, protect, show }) => {
+const Menu = ({ showOrHide, isSigned: protect, show }) => {
```

Comment:
👍 much better name, but there is no point in keeping the local variable named `protect`. Just remove it and use `isSigned` instead.